### ISSUE TITLE
Parse description field

### ISSUE THREAD

**Issue by everhardt** [sachinraja/zod-to-ts#13]
Since version 3.11.4, [zod supports descriptions](https://github.com/colinhacks/zod/issues/677). It would be great if that could be parsed as well and added to the TS-interface as inline comments.

**Comment by sachinraja** [comment]
Will look into this soon!

**Comment by everhardt** [comment]
Awesome!

**Comment by sachinraja** [comment]
Thanks! Released in v1.1.0

**Comment by Madd0g** [comment]
good stuff

I tried to add description to an object in a union of objects but it doesn't appear anywhere.

Something like:

```js
const TypeA = z.object(...).describe('my type a'); // this description is lost

const TypesColl = z.union([ TypeA, ..... ]);

const MyWrapper = z.object({
  ...
  types: z.array(TypesColl).describe('collection of types'), // this one shows up
  ... 
}); 
```

and since I'm already asking about my use-case - is there any way to give an identifier to `TypeA` and have it as a separate `type TypeA = { ... }`  in the output?
